packages/repository/src/relations/has-one/has-one.repository.ts

```diff
@@ -39,6 +40,22 @@ export interface HasOneRepository<Target extends Entity> {
     filter?: Pick<Filter<Target>, Exclude<keyof Filter<Target>, 'where'>>,
     options?: Options,
   ): Promise<Target>;
+
+  /**
+   * Delete multiple target model instances
```

Comment:
I find this confusing. In `HasMany` relation, there should be only **one** related model instance, right?

```suggestion
   * Delete the related target model instance
```